frontend/awx/resources/inventories/hooks/useInventoriesGroupsHostsToolbarActions.tsx

```diff
@@ -0,0 +1,148 @@
+            id: host.id,
+          }).then(() => void view.refresh());
+        } catch (err) {
+          alertToaster.addAlert({
+            variant: 'danger',
+            title: t(`Failed to add ${host.name} to related groups`),
+            children: err instanceof Error && err.message,
+          });
+        }
+      }
+      setDialog(undefined);
+    },
+    [setDialog, params.group_id, view, alertToaster, t]
+  );
+
+  return useMemo<IPageAction<AwxHost>[]>(
+    () => [
+      {
+        type: PageActionType.Dropdown,
+        selection: PageActionSelection.None,
+        variant: ButtonVariant.primary,
+        isPinned: true,
+        icon: PlusIcon,
+        label: t('Add host'),
+        actions: [
+          {
+            type: PageActionType.Button,
+            selection: PageActionSelection.None,
+            label: t('Existing host'),
+            onClick: () =>
+              setDialog(
+                <HostSelectDialog
+                  inventoryId={params.id as string}
+                  groupId={params.group_id as string}
+                  onSelectedHosts={onSelectedHosts}
+                />
+              ),
+          },
+          {
+            type: PageActionType.Button,
+            selection: PageActionSelection.None,
+            label: t('New group'),
+            onClick: () =>
+              pageNavigate(AwxRoute.InventoryGroupHostAdd, {
+                params: {
+                  id: params.id,
+                  inventory_type: params.inventory_type,
+                  group_id: params.group_id,
+                },
+              }),
+            isDisabled: () =>
+              canCreateHost
+                ? undefined
+                : t(
+                    'You do not have permission to create a host. Please contact your organization administrator if there is an issue with your access.'
+                  ),
+          },
+        ],
+      },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.None,
+        variant: ButtonVariant.secondary,
+        isPinned: true,
+        label: t('Run Command'),
+        onClick: () => pageNavigate(AwxRoute.Inventories),
+        isDisabled: () =>
+          view.selectedItems.length === 0
+            ? t('Select at least one item from the list')
+            : canRunAdHocCommand
+              ? undefined
+              : t(
+                  'You do not have permission to run an ad hoc command. Please contact your organization administrator if there is an issue with your access.'
+                ),
+      },
+      { type: PageActionType.Seperator },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Multiple,
+        icon: TrashIcon,
+        label: t('Disassociate selected hosts'),
+        onClick: disassociateHosts,
+        isDanger: true,
+        isDisabled:
+          view.selectedItems.length === 0 ? t('Select at least one item from the list') : undefined,
+      },
```

Comment:
Should remove the `isDanger` and trash icon and also pin to toolbar like old UI
```suggestion
      {
        type: PageActionType.Button,
        selection: PageActionSelection.Multiple,
        isPinned: true
        label: t('Disassociate selected hosts'),
        onClick: disassociateHosts,
        isDisabled:
          view.selectedItems.length === 0 ? t('Select at least one item from the list') : undefined,
      },
```